homeassistant/components/vicare/binary_sensor.py

```diff
@@ -23,8 +23,16 @@
 CONF_GETTER = "getter"
 
 SENSOR_CIRCULATION_PUMP_ACTIVE = "circulationpump_active"
+
+# gas sensors
 SENSOR_BURNER_ACTIVE = "burner_active"
+
+# heatpump sensors
 SENSOR_COMPRESSOR_ACTIVE = "compressor_active"
```

Comment:
These constants can be moved to the `const.py` file in the integration. The same for the constants in file `sensor.py`.

Reply:
I think this refactoring should be done in another PR. Since there is no const.py yet, moving all constants from all files is quite the change.